core/src/main/java/hudson/model/TimeZoneProperty.java

```diff
@@ -0,0 +1,82 @@
+package hudson.model;
+
+import hudson.Extension;
+import hudson.util.ListBoxModel;
+import hudson.util.ListBoxModel.Option;
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import javax.annotation.CheckForNull;
+import java.io.IOException;
+import java.util.TimeZone;
+
+public class TimeZoneProperty extends UserProperty implements Saveable {
+    /**
+     * Time Zone ID defined by the user.
+     * {@code null} means that the time zone is not defined.
+     */
+    @CheckForNull
+    private String timeZoneName;
+
+    @DataBoundConstructor
+    public TimeZoneProperty(@CheckForNull String timeZoneName) {
+        this.timeZoneName = timeZoneName;
+    }
+
+    private TimeZoneProperty() {
+        this(null);
+    }
+
+    public void save() throws IOException {
+        user.save();
+    }
+
+    @Extension
+    @Symbol("timeZone")
```

Comment:
```suggestion
    @Symbol("timezone")
```

No need in uppercase Z IMHO